### PR TITLE
feat: show bridge link in new network info

### PR DESCRIPTION
**Pull request by micaelae**
## **Description**
* Shows a link to the MetaMask Portfolio bridge page when the user adds a new network and switches to it. Mocks: https://docs.google.com/document/d/1x7VZpMvEtnYcR7WG7OHmBp_zp1cRLVQuMm5zskelHzU/edit
* fixes the layout for the token import bulletpoint in the new-network component so it's consistent with other list items

[Open in GitHub Codespaces](https://codespaces.new/MetaMask/metamask-extension/pull/23475?quickstart=1)

## **Related issues**
https://docs.google.com/document/d/1x7VZpMvEtnYcR7WG7OHmBp_zp1cRLVQuMm5zskelHzU/edit

## **Manual testing steps**

1. Add the Linea network and switch to it
3. Verify that "MetaMask Portfolio" link is visible
4. Click the link and it should open the Portfolio bridge page in a new tab, with the new chain as dest chain preselected
5. Add the Celo network and switch to it
6. Verify that "MetaMask Portfolio" link is not visible

### **Before**

Screenshot 2024-03-13 at 1 41 26 PM

### **After**
Screenshot 2024-03-20 at 10 25 19 AM

Screenshot 2024-03-20 at 10 26 27 AM

## **Pre-merge author checklist**

- [X] I’ve followed [MetaMask Coding Standards](https://github.com/MetaMask/metamask-extension/blob/develop/.github/guidelines/CODING_GUIDELINES.md).
- [X] I've clearly explained what problem this PR is solving and how it is solved.
- [X] I've linked related issues
- [X] I've included manual testing steps
- [X] I've included screenshots/recordings if applicable
- [X] I’ve included tests if applicable
- [X] I’ve documented my code using [JSDoc](https://jsdoc.app/) format if applicable
- [X] I’ve applied the right labels on the PR (see [labeling guidelines](https://github.com/MetaMask/metamask-extension/blob/develop/.github/guidelines/LABELING_GUIDELINES.md)). Not required for external contributors.
- [X] I’ve properly set the pull request status:
  - [ ] In case it's not yet "ready for review", I've set it to "draft".
  - [X] In case it's "ready for review", I've changed it from "draft" to "non-draft".

## **Pre-merge reviewer checklist**

- [ ] I've manually tested the PR (e.g. pull and build branch, run the app, test code being changed).
- [ ] I confirm that this PR addresses all acceptance criteria described in the ticket it closes and includes the necessary testing evidence such as recordings and or screenshots.

### CODE REVIEW

**Review by BZahory — COMMENTED** [review]
Let's exclude networks that Bridging doesn't support – use `ALLOWED_BRIDGE_CHAIN_IDS`

Screenshot 2024-03-14 at 5 03 36 PM

**Review by BZahory — DISMISSED** [review]

**Review by BZahory — DISMISSED** [review]

File: ui/components/ui/new-network-info/new-network-info.test.js [hunk 1]
```diff
@@ -110,13 +110,13 @@ describe('NewNetworkInfo', () => {
...
       // render title
-      expect(getByText('You have switched to')).toBeInTheDocument();
+      expect(getByText("You're now using")).toBeInTheDocument();
       // render the network name
       expect(getByText('Ethereum Mainnet')).toBeInTheDocument();
       expect(
         getByTestId('new-network-info__bullet-paragraph').textContent,
       ).toMatchInlineSnapshot(
-        `"• The native token on this network is ETH. It is the token used for gas fees. "`,
+        `"Gas is ETHThe native token on this network is ETH. It is the token used for gas fees."`,
```
Comment (DDDDDanica) [inline ui/components/ui/new-network-info/new-network-info.test.js:119]: it seems like we are missing a space here :)

File: ui/components/ui/new-network-info/new-network-info.js [hunk 2]
```diff
@@ -63,151 +72,182 @@ export default function NewNetworkInfo() {
...
-                  &bull;
+                <Box className="new-network-info__bullet-icon-container">
+                  <Icon name={IconName.Coin} size={IconSize.Sm} />
                 </Box>
-                <Box>
+                <Box flexDirection={FlexDirection.Column}>
+                  <Text
+                    variant={TextVariant.bodySmBold}
+                    as="h6"
+                    color={Color.textDefault}
```
Comment (DDDDDanica) [inline ui/components/ui/new-network-info/new-network-info.js:239]: Nit: 
```suggestion
                    color={TextColor.textDefault}
```

File: ui/components/ui/new-network-info/new-network-info.js [hunk 3]
```diff
@@ -63,151 +72,182 @@ export default function NewNetworkInfo() {
...
+                            metaMetricsId,
+                          )}&destChain=${currentNetwork?.chainId}`}
+                          target="_blank"
+                          rel="noreferrer"
+                          key="bridge-link"
+                        >
+                          <Text
+                            variant={TextVariant.bodySm}
+                            as="h6"
+                            color={Color.infoDefault}
```
Comment (DDDDDanica) [inline ui/components/ui/new-network-info/new-network-info.js:211]: Nit: 
```suggestion
                            color={TextColor.infoDefault}
```

File: ui/components/ui/new-network-info/new-network-info.js [hunk 4]
```diff
@@ -63,151 +72,182 @@ export default function NewNetworkInfo() {
...
-                    {t('learnMoreUpperCase')}
-                  </Text>
-                </a>
-              </Text>
+                  {t('bridgeDontSend')}
+                </Text>
+                <Text
+                  variant={TextVariant.bodySm}
+                  as="h6"
+                  color={Color.textDefault}
```
Comment (DDDDDanica) [inline ui/components/ui/new-network-info/new-network-info.js:193]: Nit: 
```suggestion
                  color={TextColor.textDefault}
```

File: ui/components/ui/new-network-info/new-network-info.js [hunk 5]
```diff
@@ -63,151 +72,182 @@ export default function NewNetworkInfo() {
...
-                {t('attemptSendingAssets')}{' '}
-                <a
-                  href="https://metamask.zendesk.com/hc/en-us/articles/4404424659995"
-                  target="_blank"
-                  rel="noreferrer"
+              <Box flexDirection={FlexDirection.Column}>
+                <Text
+                  variant={TextVariant.bodySmBold}
+                  as="h6"
+                  color={Color.textDefault}
```
Comment (DDDDDanica) [inline ui/components/ui/new-network-info/new-network-info.js:186]: Nit:
```suggestion
                  color={TextColor.textDefault}
```

File: ui/components/ui/new-network-info/new-network-info.js [hunk 6]
```diff
@@ -63,151 +72,182 @@ export default function NewNetworkInfo() {
...
+                    variant={TextVariant.bodySmBold}
+                    as="h6"
+                    color={Color.textDefault}
+                  >
+                    {t('gasIsETH', [providerConfig.ticker])}
+                  </Text>
+                  <Text
+                    variant={TextVariant.bodySm}
+                    as="h6"
+                    color={Color.textDefault}
```
Comment (DDDDDanica) [inline ui/components/ui/new-network-info/new-network-info.js:163]: Nit
```suggestion
                    color={TextColor.textDefault}
```

File: ui/components/ui/new-network-info/new-network-info.js [hunk 7]
```diff
@@ -63,151 +72,182 @@ export default function NewNetworkInfo() {
...
-                <Box marginRight={4} color={Color.textDefault}>
-                  &bull;
+                <Box className="new-network-info__bullet-icon-container">
+                  <Icon name={IconName.Gas} size={IconSize.Sm} />
+                </Box>
+                <Box flexDirection={FlexDirection.Column}>
+                  <Text
+                    variant={TextVariant.bodySmBold}
+                    as="h6"
+                    color={Color.textDefault}
```
Comment (DDDDDanica) [inline ui/components/ui/new-network-info/new-network-info.js:156]: Nit: 
```suggestion
                    color={TextColor.textDefault}
```

File: ui/components/ui/new-network-info/new-network-info.js [hunk 8]
```diff
@@ -63,151 +72,182 @@ export default function NewNetworkInfo() {
...
+            <Button
+              variant="primary"
+              onClick={onCloseClick}
+              size={ButtonPrimarySize.Md}
+              className="footer__button"
+            >
+              <Text
+                variant={TextVariant.bodySm}
+                as="h6"
+                color={Color.textDefault}
```
Comment (DDDDDanica) [inline ui/components/ui/new-network-info/new-network-info.js:103]: Nit:
```suggestion
                color={TextColor.textDefault}
```

File: ui/components/ui/new-network-info/new-network-info.js [hunk 9]
```diff
@@ -63,151 +72,182 @@ export default function NewNetworkInfo() {
...
         className="new-network-info__wrapper"
+        headerProps={{ marginLeft: 6 }}
         footer={
-          <Button type="primary" onClick={onCloseClick}>
-            {t('recoveryPhraseReminderConfirm')}
-          </Button>
+          <>
+            <Button
+              variant="secondary"
+              href="https://metamask.zendesk.com/hc/en-us/articles/4404424659995"
```
Comment (DDDDDanica) [inline ui/components/ui/new-network-info/new-network-info.js:84]: Nit: Can you move the link to `ui/helpers/constants/zendesk-url.js`?

Comment (micaelae) [reply ui/components/ui/new-network-info/new-network-info.js:84]: good call, it was already in there so I'll reuse the constant

**Review by ejwessel — DISMISSED** [review]

**Review by BZahory — DISMISSED** [review]

**Review by DDDDDanica — DISMISSED** [review]

**Review by BZahory — APPROVED** [review]

**Review by DDDDDanica — APPROVED** [review]

**Pull request merged**